styles/output/nursing-internal-pdf.css

```diff
@@ -1807,104 +1807,45 @@ h4.os-subtitle + .os-figure:not(.has-splash) > figure {
   margin-left: auto;
   margin-right: auto;
 }
-
 .os-table:not(.os-unstyled-container) > table {
   border-collapse: collapse;
   width: 100%;
   margin-bottom: 0;
 }
-
 .os-table:not(.os-unstyled-container) > table > thead > tr > th {
-  border-bottom: 0.1rem solid black;
-  border-top: 0.1rem solid black;
+  border-bottom: 0.1rem solid #000000;
+  border-top: 0.1rem solid #000000;
   font-family: Mulish, sans-serif;
   font-weight: 700;
   padding: 0.7rem;
 }
-
 .os-table:not(.os-unstyled-container) > table > thead > tr > th[data-valign=top] {
   vertical-align: top;
 }
-
 .os-table:not(.os-unstyled-container) > table > thead > tr > th[data-align=left] {
   text-align: left;
 }
-
 .os-table:not(.os-unstyled-container) > table > thead > tr > th[data-align=right] {
   text-align: right;
 }
-
 .os-table:not(.os-unstyled-container) > table > thead > tr > th[data-align=center] {
   text-align: center;
 }
-
-.os-table:not(.os-unstyled-container) > table > tbody > tr.header-row > td {
-  border-top-color: black;
-  border-top-width: 0.1rem;
-  border-top-style: solid;
-  border-bottom-color: black;
-  border-bottom-width: 0.1rem;
-  border-bottom-style: solid;
-  padding-top: 0.7rem;
-  padding-bottom: 0.7rem;
-  font-weight: 700;
-  padding-left: 0.7rem;
-  padding-right: 0.7rem;
-}
-
 .os-table:not(.os-unstyled-container) > table > tbody > tr > td[data-valign=top] {
   vertical-align: top;
 }
-
 .os-table:not(.os-unstyled-container) > table > tbody > tr > td[data-align=left] {
   text-align: left;
 }
-
 .os-table:not(.os-unstyled-container) > table > tbody > tr > td[data-align=right] {
   text-align: right;
 }
-
 .os-table:not(.os-unstyled-container) > table > tbody > tr > td[data-align=center] {
   text-align: center;
 }
 
-.os-table:not(.os-unstyled-container) > table > tbody > tr > td:not(:only-of-type) {
```

Comment:
i did a ctrl + F for this selector and couldn't find it in the new output anywhere ... it's hard to see because things have moved around, but think it's disappeared

Reply:
Thanks for catching this selector was nested in the wrong place. I've updated this and can now see the selector in the output file.